### PR TITLE
Adds SonarCloud

### PR DESCRIPTION
**Pull request by victormendes5**
### Description

- Integrates SonarCloud to the project;
- Adds Bitrise and SonarCloud badges;
- Changes `CONTRIBUTION` emoji description;
- Removes SonarCloud Bugs;
- Updates gems;
- Adds new `sonar_scanner` lane to fastlane;

#### Current code coverage: **89,7**%

### CODE REVIEW

File: EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift [hunk 1]
```diff
@@ -81,28 +81,28 @@ extension UIColor {
         
         public struct Light {
             /// #FFFFFF - 255, 255, 255
-            public static let white1 = UIColor(red: 255/255, green: 255/255, blue: 255/255, alpha: 1.0)
```
Comment (Luan-BK) [inline EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift]: Is it not necessary to divide these values by 255 anymore?

File: fastlane/Pipelines/TestsFastfile [hunk 2]
```diff
@@ -47,11 +50,27 @@ lane :coverage do
...
 end
 
 desc "Run UITests and get screenshots"
 lane :screenshots do
     capture_screenshots
 end
+
+desc "Convert xccov to xml"
+lane :sonnar_coverage do
+    # sh "chmod +x xccov-to-sonarqube-generic.sh"
```
Comment (Luan-BK) [inline fastlane/Pipelines/TestsFastfile]: Is this commented on purpose?

File: EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift (hunk 1 above)
Comment (victormendes5) [reply EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift]: No, SonarCloud was detected it as a bug

File: fastlane/Pipelines/TestsFastfile (hunk 2 above)
Comment (victormendes5) [reply fastlane/Pipelines/TestsFastfile]: Yep, I need to do a test with it

File: EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift (hunk 1 above)
Comment (victormendes5) [reply EmeraldComponents/Extensions/UIColor/UIColor Extensions/UIColor+Palette.swift]: Sorry, you're right! SonarCloud is crazy, I believe I need to config it better.

**Pull request merged**